transformers/shared/constants/keys.go

```diff
@@ -54,6 +54,9 @@ const (
 	DataColumn    event.ColumnName = "data"
 	InkColumn     event.ColumnName = "ink"
 	ArtColumn     event.ColumnName = "art"
+	RadColumn     event.ColumnName = "rad"
 	TabColumn     event.ColumnName = "tab"
+	UColumn       event.ColumnName = "u"
```

Comment:
@elizabethengelman Are we adding these here now? Was looking at your PR...

Reply:
Yep!